bokehjs/src/less/attribution.less

```diff
@@ -3,8 +3,12 @@
   flex-direction: column;
   flex-wrap: nowrap;
   padding: 2px;
+  // @mattpap: So I think the 0.5 alpha value is to help this look good no
+  // matter what background color is used for the data viz, but this creates
+  // contrast issues when the background is dark, and text color of the
+  // attribution is also dark. Not sure what to do about this. Any thoughts?
   background-color: rgba(255, 255, 255, 0.5);
-  font-size: 9px;
+  font-size: 0.5625rem; // 9px if base font is 16px
```

Comment:
Wouldn't it be better to configure this as a percentage of the base font?

Reply:
I suppose both approaches are valid in certain applications and as long as used consistently. However, given that we indeed define everything in terms of the base font size, then perhaps it would be better to use `rem` units.